### PR TITLE
Add environment field to rule

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

File: rulesengine/rules/migrations/0010_auto_20180912_1524.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
+# Generated by Django 2.1 on 2018-09-12 15:24
```
Comment (makyo) [inline rulesengine/rules/migrations/0010_auto_20180912_1524.py]: Autogenerated - can disregard

**Comment by BitBaron** [comment]
Our existing Rules Engine doesn't keep track of environments. We simply plug it into a different DB to store it's values.

I'd prefer to have the application be environment agnostic unless there's some specific requirement.

**Comment by makyo** [comment]
@BitBaron Ahh, I understand. I think @jkafader may have more info, but it sounded like we might be giving rules-editing to a wider audience, including contacts with partners, in which case they might want the ability to QA rules before applying them globally. I'll clarify before going anywhere with this.

File: rulesengine/rules/utils/validators.py [hunk 2]
```diff
@@ -12,11 +12,18 @@
     ('rewrite-headers', 'Rewrite playback headers'),
 )
 
+ENVIRONMENT_CHOICES = (
+    ('prod', 'Production'),
+    ('qa', 'QA/development'),
```
Comment (jkafader) [inline rulesengine/rules/utils/validators.py]: I think we should probably name this e.g. `"test", "Testing/deveopment"` -- we have too many things named "QA" already.

**Pull request merged**